README.md

```diff
@@ -1,27 +1 @@
-## Homework #2 css responsive
```

Comment:
can you not modify readme? - you can merge master into your branch locally and accept changes from master branch for this file

Reply:
Whoops. Will do that way next time!